packages/web3-eth-accounts/src/account.ts

```diff
@@ -158,8 +250,12 @@ export const signTransaction = (
 
 /**
  * Recovers the Ethereum address which was used to sign the given RLP encoded transaction.
- *
- * @param rawTransaction
+ * @param rawTransaction - The RLP encoded transaction
```

Comment:
```suggestion
 * @param rawTransaction - The hex string having RLP encoded transaction
```

I think with more detail?